safe/impact_reports/population_exposure_report_mixin.py

```diff
@@ -0,0 +1,290 @@
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool developed by AusAid -
+**Building Exposure Report Mixin Class**
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+"""
+__author__ = 'Christian Christelis <christian@kartoza.com>'
+__revision__ = '$Format:%H$'
+__date__ = '05/05/2015'
+__copyright__ = ('Copyright 2012, Australia Indonesia Facility for '
+                 'Disaster Reduction')
+
+from collections import OrderedDict
+
+from safe.utilities.i18n import tr
+from safe.common.utilities import format_int
+from safe.impact_reports.report_mixin_base import ReportMixin
+
+from safe.impact_functions.core import (
+    evacuated_population_needs,
+    population_rounding)
+
+
+class PopulationExposureReportMixin(ReportMixin):
+    """Building specific report.
```

Comment:
It would be nice to include a general description of the expected usage pattern here